.github/workflows/pkgci_build_packages.yml

```diff
@@ -15,12 +15,7 @@ on:
 jobs:
   linux_x86_64_release_packages:
     name: Linux Release (x86_64)
-    runs-on:
-      - self-hosted # must come first
-      - runner-group=${{ github.event_name == 'pull_request' && 'presubmit' || 'postsubmit' }}
-      - environment=prod
-      - cpu
-      - os-family=Linux
+    runs-on: azure-linux-scale
```

Comment:
You had some docs for runner configuration. Are those public or can parts of them be public? We could link to them from here: https://iree.dev/developers/general/github-actions/#self-hosted-runner-maintenance

Reply:
It's all public, but I have it on my own repo here: https://github.com/saienduri/AKS-GitHubARC-Setup. Not sure if we want to add a link to my repo into the iree docs or create another repo?

Reply:
A link to there is fine as a start. If the docs are small enough and generally useful to the project, we can add them in-tree. If the docs (and images used, etc.) get large, we may want to consider moving the docs/ directory to a full standalone repo like https://github.com/llvm/llvm-www. I like being able to update docs alongside code though.